src/App.css

```diff
@@ -1,38 +1,21 @@
 .App {
   text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
+  background-color: #282c34;
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+  /* color: white; */
```

Comment:
- [ ] Please remove commented lines.

Reply:
Yes @Yothu, I going to make these changes :computer:, thank you!